test/integration/exchangeIssuanceZeroExIntegration.spec.ts

```diff
@@ -0,0 +1,442 @@
+import "module-alias/register";
```

Comment:
This is an integration test script that is meant to be run manually against a local mainnet fork.

Since it uses the response from the real 0x api it needs to be executed against a fork that is as "up to date" with the latest block as possible. 

To run these tests you should:
1. Start local mainnet fork with `FORK=true LATESTBLOCK=true yarn chain`
2. Run integrationtest with `INTEGRATIONTEST=true VERBOSE=true yarn test test/integration/exchangeIssuanceZeroExIntegration.spec.ts` (leave out verbosity flag if you don't want to see details on the 0x api requests / response)

Reply:
This test can also be used as inspiration for building a frontend integration.